src/main/java/core/basesyntax/dao/StorageDaoImpl.java

```diff
@@ -9,13 +9,16 @@ public class StorageDaoImpl implements StorageDao {
     @Override
     public User add(User user) {
         user.setId(++index);
-        Storage.people.add(user);
+        Storage.PEOPLE.add(user);
         return user;
     }
 
     @Override
     public User get(String login) {
-        for (User user : Storage.people) {
+        if (login == null) {
+            throw new NullPointerException("Login can't be null");
+        }
```

Comment:
I don't think you should modify this method

in addition - your `register` method should verify the login before storing a user